python/examples/handler.py

```diff
@@ -45,10 +39,7 @@ def say_ok(request, response, proxy):
     yield print_arg(request, 1)
     yield print_arg(request, 2)
 
-    response.argstreams = [
-        InMemStream(),
-        InMemStream(),
-        InMemStream("world")]
+    response.set_body(InMemStream("world"))
```

Comment:
we also have pipestream. stream a file. we need to provide the ability to support that. We can find a better name